learning_resources/serializers.py

```diff
@@ -0,0 +1,127 @@
+"""Serializers for learning_resources"""
+from rest_framework import serializers
+
+from learning_resources import models
+
+COMMON_IGNORED_FIELDS = ("created_on", "updated_on")
+
+
+class LearningResourceInstructorSerializer(serializers.ModelSerializer):
+    """
+    Serializer for LearningResourceInstructor model
+    """
+
+    class Meta:
+        model = models.LearningResourceInstructor
+        fields = "__all__"
```

Comment:
Should we exclude `COMMON_IGNORED_FIELDS` here, too?

_I don't object to the created/updated fields, just noticed them in the API results and they are excluded elsewhere_